src/test/groovy/com/stormpath/tck/util/CookieUtils.groovy

```diff
@@ -22,6 +22,6 @@ class CookieUtils {
 
     public static boolean isCookieDeleted(Cookie cookie) {
         def now = new Date()
-        return cookie.expiryDate < now
+        return cookie == null || cookie.expiryDate < now
```

Comment:
If there is no cookie present at all, such as on an initial request, the cookie will be null.